CommonLibSF/include/SFSE/Utilities.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "REL/Relocation.h"
+#include "SFSE/API.h"
+
+namespace SFSE
+{
+	namespace stl
+	{
+		template <class T, std::size_t Size = 5>
+		constexpr void write_thunk_call() noexcept
+		{
+			SFSE::AllocTrampoline(14);
+			auto& trampoline{ SFSE::GetTrampoline() };
+			T::func = trampoline.write_call<Size>(T::address, T::Thunk);
+		}
+
+		template <class TDest, class TSource>
+		constexpr void write_vfunc() noexcept
+		{
+			REL::Relocation vtbl{ TDest::VTABLE[0] };
```

Comment:
Specify `REL::Relocation<std::uintptr_t>` in line with other REL::Relocation usage

Reply:
Doesn't CTAD take care of that? Added it anyway